LicenseHeaderManager/LicenseHeaderManager.csproj

```diff
@@ -79,7 +81,8 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <DeployExtension>True</DeployExtension>
+    <DeployExtension>False</DeployExtension>
```

Comment:
```suggestion
    <DeployExtension>True</DeployExtension>
```

Unnecessary change

Reply:
Fixed in https://github.com/rubicon-oss/LicenseHeaderManager/pull/104/commits/3376d23e852fc8036c284627aa3a2145e9c618c5